meow/eme/sax.py

```diff
@@ -32,7 +32,7 @@ def _get_netlist(propagations, interfaces):
         **{k: S for k, S in interfaces.items()},
     }
 
-    interface_keys = list(interfaces)
+    list(interfaces)
```

Comment:
This line can also be omitted

Reply:
I don't understand why this code 'improvement' tool not automatically removes the full line.